### PR TITLE
refactor Download model highest_count method

### PR DESCRIPTION
**Pull request by farukaydin**
used ```rank(version)``` instead of ```Download.rank(version)``` and ternary operator for single-line condition

cc: @sferik

### CODE REVIEW

File: app/models/download.rb [hunk 1]
```diff
@@ -212,12 +212,8 @@ def self.rank(version)
...
 
   def self.highest_rank(versions)
-    ranks = versions.map { |version| Download.rank(version) }.reject(&:zero?)
-    if ranks.empty?
-      0
-    else
-      ranks.min
-    end
+    ranks = versions.map { |version| rank(version) }.reject(&:zero?)
+    ranks.empty? ? 0 : ranks.min
```
Comment (qrush) [inline app/models/download.rb]: What a weird section of code. We reject all entries that are zero - but then if we don't find any, zero is the default. Can we just use `min { |version| rank(version)` ?  Also `#min` returns nil if there's nothing in the array, so we don't need the ternary at all.

Comment (farukaydin) [inline app/models/download.rb]: @qrush :smiley: I searched a little bit in git history and first version of this method was like you said  . I don't know but maybe we just converted it to this form for specific needs. Anyway, when I was searching, I find out that this method isn't used anymore( via https://github.com/rubygems/rubygems.org/commit/9c284621a008fc69c58844cdf3beb34a8e00e0a3 ) I'm planning to delete the method and it's test, and rebased commits to one. Sounds good?

Comment (qrush) [inline app/models/download.rb]: :gun: :+1:

**Comment by farukaydin** [comment]
@qrush updated.

**Comment by qrush** [comment]
ALL RED!!! :tada: :ok: :ship: :gun: :pray: :100:

Can you remove the `nl` translation too?

```
config/locales/nl.yml
44:      highest_rank: hoogste notering vandaag
```

**Comment by farukaydin** [comment]
@qrush it's done. :ok:

**Pull request merged**